src/configureStore.test.js

```diff
@@ -29,6 +28,32 @@ describe('configureStore', () => {
     it('should contain a hook for `sagaMiddleware.run`', () => {
       expect(typeof store.runSaga).toBe('function');
     });
+
+    it('should catch errors thrown from sagas', () => {
+      const dispatchSpy = jest.spyOn(store, 'dispatch');
+      const title = 'Whoopsie';
+      const error = new Error(title);
+
+      function* someGenerator() {
+        throw error;
+      }
+
+      expect(dispatchSpy).not.toHaveBeenCalled();
+
+      store.runSaga(someGenerator);
+
+      expect(dispatchSpy).toHaveBeenCalledWith(
+        expect.objectContaining({
+          payload: {
+            message: expect.any(String),
```

Comment:
We know the message and the string at this point right?

Reply:
Yes, but for the test it doesn't really matter which message has been sent, just that a message has been sent.